ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -283,6 +481,22 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-ipsec
+- as: gcp-ipi-proxy-etcd-encryption-f28-compliance
```

Comment:
`compliance` seems to me a special e2e test. If we really want to cover this profile, we can remove `compliance` from the job name and then remove related env variables. 
cc @jianlinliu to confirm.

Reply:
removed the job.